terraform-vpc/securitygroup.tf

```diff
@@ -0,0 +1,97 @@
+resource "aws_security_group" "website" {
+    name = "website"
+    description = "Allow inbound traffic"
+    vpc_id = "${aws_vpc.da-wordpress-vpc.id}"
+
+    tags {
+        Name = "website"
+    }
+
+    ingress {
+        from_port = 22
+        to_port = 22
+        protocol = "tcp"
+        cidr_blocks = ["0.0.0.0/0"]
+    }
+
+    ingress {
+        from_port = 80
+        to_port = 80
+        protocol = "tcp"
+        cidr_blocks = ["0.0.0.0/0"]
+    }
+
+    ingress {
+        from_port = 443
+        to_port = 443
+        protocol = "tcp"
+        cidr_blocks = ["0.0.0.0/0"]
+    }
+
+    egress {
+        from_port = 0
+        to_port = 0
+        protocol = "-1"
+        cidr_blocks = ["0.0.0.0/0"]
+    }
+
+}
+
+resource "aws_security_group" "database" {
+    name = "database"
+    description = "Allow inbound traffic"
+    vpc_id = "${aws_vpc.da-wordpress-vpc.id}"
+
+    tags {
+        Name = "database"
+    }
+
+    ingress {
+        from_port = 22
+        to_port = 22
+        protocol = "tcp"
+        security_groups  = ["${aws_security_group.website.id}"]
+    }
+
+    ingress {
+        from_port = 3306
+        to_port = 3306
+        protocol = "tcp"
+        security_groups  = ["${aws_security_group.website.id}"]
+    }
```

Comment:
What service will be used and why are you already creating something that is not being created in this PR?

Reply:
I tried to make the workflow more efficiently by having the DB Security group in place first in the VPC and then making the DB Piece fit in to here.

Reply:
Don't create resources for the future PRs. It makes the review harder and slow down the `merging to master` process.